Administrator Guide/arbiter-volumes-and-quorum.md

```diff
@@ -0,0 +1,219 @@
+    necessary that the first brick is one of the up bricks.
+    
+>Option: cluster.quorum-count  
+>Value Description:                                                                 
+    The number of bricks that must be active in a replica-set to allow writes.
+    This option is used in conjunction with cluster.quorum-type *=fixed* option
+    to specify the number of bricks to be active to participate in quorum.
+    If the quorum-type is auto then this option has no significance.
+                                                       
+> Option: cluster.quorum-reads  
+Default Value: no  
+Value Description: yes|no
+If quorum-reads is set to 'yes' (or 'true' or 'on') then even reads will be allowed
+only if quorum is met, without which the read (and writes) will return ENOTCONN.
+If set to 'no' (or 'false' or 'off'), then reads will be served even when quorum is
+not met, but writes will fail with EROFS.
+
+
+Replica 2 and Replica  3 volumes
+===============================
+From the above descriptions, it is clear that client-quorum cannot really be applied
+to a replica 2 volume:(without costing HA).
+If the quorum-type is set to auto, then by the description
+given earlier, the first brick must always be up, irrespective of the status of the
+second brick. IOW, if only the second brick is up, the  subvol becomes EROFS, i.e. no HA.
+If quorum-type is set to fixed, the the quorum-count *has* to be two
+to prevent split-brains. (otherwise a write can succeed in brick1, another in brick2 =>split-brain).
+So for all practical purposes, if you want high availabilty in a replica 2 volume,
+it is recommended not enable client-quorum.
+
+In a replica 3 volume, client-quorum is enabled by default and set to 'auto'.
+This means 2 bricks need to be up for the writes to succeed. Here is how this
+configuration prevents files from ending up in split-brain:
+
+Say B1, B2 and B3 are the bricks:
+1. B3 is down, quorum is met, write happens on  the file on B1 and B2
+2. B3 comes up, B2 is down, quorum is again met, a write happens on B1 and B3.
+3. B2 comes up, B1 goes down, quorum is met. Now when a write is issued, AFR sees
+that B2 and B3's pending xattrs blame each other and therefore the write is not
+allowed and is failed with an EIO.
+
+There is a corner case even with replica 3 volumes where the file can
+end up in a split-brain. AFR usually takes range locks for the {offset, length}
+of the write. If 3 writes happen on the same file at non-overlapping {offset, length}
+and each write fails on (only) one different brick, then we have AFR xattrs of the
+file blaming each other.
+
+Arbiter configuration
+======================
+The arbiter configuration a.k.a. the arbiter volume is the perfect sweet spot
+between a 2-way replica and 3-way replica to avoid files getting into split-brain.
+The syntax for creating the volume is:
+>#gluster volume create <VOLNAME>  replica 3 arbiter 1 host1:brick1 host2:brick2 host3:brick3
```

Comment:
This is rendered in a huge font because of `#`. Consider using `~~~` at opening and closing. I use [http://markable.in/](http://markable.in/) to check rendering